Units/clojure.d/input.clj

```diff
@@ -0,0 +1,10 @@
+(ns app.controller)
+
+(defn empty-fn [])
+
+(defn function-with-body []
+    (println "body"))
+
+'(defn quoted-function [])
+(quote quoted-function2 [])
```

Comment:
shouldn't `quoted-function` and `quoted-function2` appear in the tags?  (just assuming, I actually can't read clojure :)